ui/imports/shared/controls/TransactionAddress.qml

```diff
@@ -0,0 +1,134 @@
+import QtQuick 2.15
```

Comment:
I wonder why we couldn't reuse StatusListItem for this, maybe you have a thought behind this already